test/test_transmon_expanded.py

```diff
@@ -260,23 +260,23 @@ def test_propagation():
     np.testing.assert_allclose(
         actual=test_data["propagators_q1"],
         desired=data["propagators_q1"],
-        atol=1e-11 * np.max(data["propagators_q1"]),
+        atol=1e-12,
```

Comment:
Do we want to add a note on why some of the `atol` values are `1e-8` and some are `1e-12` (also use a global constant for `1e-12`). This might be useful for later debugging to document your present thought process.